### PR TITLE
Adds some source-tree tests

### PR DESCRIPTION
**Pull request by lukaszsobek**
Adds source-tree reducer tests

### CODE REVIEW

File: src/reducers/tests/source-tree.spec.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+  it("updates state", () => {
+    const expanded = "updateTest";
+    const state = InitialState();
+    const action = {
+      type: "SET_EXPANDED_STATE",
+      expanded
+    };
+    const updatedState = update(state, action);
+    expect(updatedState.toJS()).toEqual({ expanded });
```
Comment (lukaszsobek) [inline src/reducers/tests/source-tree.spec.js]: should be just `expect(updatedState).toEqual(state);`

File: src/reducers/tests/source-tree.spec.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+ * file, You can obtain one at <http://mozilla.org/MPL/2.0/>. */
+
+import update, { InitialState, getExpandedState } from "../source-tree";
+
+describe("source-tree", () => {
+  it("creates initial state", () => {
+    const state = InitialState();
+    const action = { type: "@@INIT" };
+    const updatedState = update(state, action);
+    expect(updatedState.toJS()).toEqual({
```
Comment (lukaszsobek) [inline src/reducers/tests/source-tree.spec.js]: should be just `expect(updatedState).toEqual(state);`

**Comment by jasonLaster** [comment]
we actually dont write reducer tests because they're in theory tested by the action tests
https://github.com/devtools-html/debugger.html/blob/master/src/actions/tests/source-tree.spec.js#L18

could you check here and see if there are any missing tests?

**Comment by darkwing** [comment]
Closing this per @jasonLaster's comments, but please feel free to reopen if you feel appropriate @lukaszsobek !

**Pull request closed without merging**